src/_includes/social.html

```diff
@@ -0,0 +1,19 @@
+<nav class="social">
```

Comment:
can this file be included in the `footer.html` so there's less repetition of the same code throughout the codebase?